static/js/components/CustomInput.jsx

```diff
@@ -0,0 +1,51 @@
+import { TextField } from "@material-ui/core";
```

Comment:
We have standardized on the following input order (I'm surprised eslint doesn't warn about this!):

- React imports
- Material + all other library imports
- Own imports, such as from baselayer, and styles.